src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -96,25 +97,70 @@ public static AminoAcid getEukaryoticAminoAcidByCodon(final String codon) {
     }
 
     /**
-     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code codon}.
+     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code rawCodon}.
      * The codons given are expected to be valid for Mitochondrial DNA.
-     * @param codon The three-letter codon (each letter one of A,T,G,C) representing a Mitochondrial {@link AminoAcid}
-     * @return The {@link AminoAcid} corresponding to the given {@code codon}.  Returns {@code null} if the given {@code codon} does not code for a Mitochondrial {@link AminoAcid}.
+     * Assumes no special cases for alternate initiation sites.
+     * @param rawCodon The three-letter codon (each letter one of A,[T or U],G,C) representing a Mitochondrial {@link AminoAcid}
+     * @return The {@link AminoAcid} corresponding to the given {@code rawCodon}.  Returns {@code null} if the given {@code rawCodon} does not code for a Mitochondrial {@link AminoAcid}.
      */
-    public static AminoAcid getMitochondrialAminoAcidByCodon(final String codon, final boolean isFirst) {
+    public static AminoAcid getMitochondrialAminoAcidByCodon(final String rawCodon) {
+        return getMitochondrialAminoAcidByCodon(rawCodon, false, false, false, false, false);
```

Comment:
Why are all of these false?

Reply:
Because with this overload there is no way to determine what the genus is and if the codon is first in the sequence.

In practice this is OK according to Sarah Calvo: 

I would just code the vertebrate mito codon as the standard codon table with the 4 differences:

AGA | Ter * | Arg R
-- | -- | --
AGG | Ter * | Arg R
AUA | Met M | Ile I
UGA | Trp W | Ter *

So, AUA and AUG code for Methionine (and AUU should still code for Isoleusine).